src/components/views/messages/MLocationBody.tsx

```diff
@@ -168,7 +168,7 @@ export const LocationBodyFallbackContent: React.FC<{ event: MatrixEvent, error:
 };
 
 export function LocationBodyContent(props: ILocationBodyContentProps):
-        React.ReactElement<HTMLDivElement> {
+    React.ReactElement<HTMLDivElement> {
```

Comment:
style guide wants the function to be like this:

```ts
export function LocationBodyContent(
    props: ILocationBodyContentProps,
): React.ReactElement<HTMLDivElement> {
    // ...
}
```

also probably shouldn't be NamedLikeThis, but happy to concede on that point for now.

Reply:
changed to use React.FC instead